.changelog/0.9.0.toml

```
@@ -3,13 +3,15 @@ title = ""
 description = ""
 
 [[features]]
-title = ""
-description = ""
+title = "New resource"
+description = "`oxide_floating_ip` [#427](https://github.com/oxidecomputer/terraform-provider-oxide/pull/427)."
+title = "New data resource"
+description = "`oxide_floating_ip` [#427](https://github.com/oxidecomputer/terraform-provider-oxide/pull/427)."
```

Argh, so sorry I missed this! This isn't valid toml 😅 .

```suggestion
title = "New resource"
description = "`oxide_floating_ip` [#427](https://github.com/oxidecomputer/terraform-provider-oxide/pull/427)."

[[features]]
title = "New data resource"
description = "`oxide_floating_ip` [#427](https://github.com/oxidecomputer/terraform-provider-oxide/pull/427)."
```

---

I always forget arrays of tables are like this. Fixed!